src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
+      </h2>
+      {age && (
```

Comment:
if `age === 0` then the `0` will be displayed before the `p` tag.
```suggestion
      {age !== undefined && (
 ```

Reply:
The thing with {' '} is not my idea, it's just the way to get rid of linter...